ui/src/elements/ngm-auth.ts

```diff
@@ -63,7 +63,7 @@ export class NgmAuth extends LitElementI18n {
       <div class="ngm-user ${classMap({'ngm-active-section': !!this.user})}"
            @click=${!this.user ? this.login : this.logout}>
         <div class="ngm-user-icon"></div>
-        ${!this.user ? i18next.t('lsb_login') : i18next.t('lsb_logout')}
+        <!-- ${!this.user ? i18next.t('lsb_login') : i18next.t('lsb_logout')} -->
```

Comment:
The "Login" text is hidden now. Is it expected?

Reply:
@vladyslav-tk 
Yes, that's correct. As for other applications, the log button does not show a text label.

Reply:
Done